src/analysis/normed_space/star/complex.lean

```diff
@@ -0,0 +1,57 @@
+/-
+Copyright (c) 2022 Frédéric Dupuis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Frédéric Dupuis
+-/
+
+import analysis.normed_space.star.basic
+import algebra.star.module
+import analysis.complex.basic
+
+/-!
+# Complex normed star modules and algebras
+
+Facts about star modules and star algebras over the complex numbers.
+
+## Main definitions
+
+* `star_module.mul_neg_I_lin`: multiplication by -I as a real-linear equivalence between the
+  skew-adjoint and self-adjoint elements of a star module.
+* `star_module.im`: the imaginary part of an element of a star module, defined via
+  `skew_adjoint_part`.
+
+-/
+
+variables {E : Type*}
+
+namespace star_module
+open_locale complex_conjugate
+open complex
+
+variables [add_comm_group E] [star_add_monoid E] [module ℂ E] [star_module ℂ E]
+
+/-- Multiplication by -I as a real-linear equivalence between the skew-adjoint and self-adjoint
+elements of a star module. -/
+@[simps] def mul_neg_I_lin : skew_adjoint E ≃ₗ[ℝ] self_adjoint E :=
+{ to_fun := λ x, ⟨-I • x, by simp [self_adjoint.mem_iff]⟩,
+  inv_fun := λ x, ⟨I • x, by simp [skew_adjoint.mem_iff]⟩,
+  map_add' := λ x y, by { ext, simp only [add_subgroup.coe_add, smul_add, add_subgroup.coe_mk] },
+  map_smul' := λ r x, by { ext, simp only [neg_smul, neg_inj, skew_adjoint.coe_smul,
+                add_subgroup.coe_mk, ring_hom.id_apply, self_adjoint.coe_smul, smul_neg],
+                rw [smul_comm] },
```

Comment:
You can push lemmas that would normally loop inside `simp` by specialising them like this:
```suggestion
  map_smul' := λ r x, by { ext, simp only [neg_smul, neg_inj, skew_adjoint.coe_smul,
    add_subgroup.coe_mk, ring_hom.id_apply, self_adjoint.coe_smul, smul_neg, smul_comm I], },
```

Reply:
Right! I should try that trick more often...